src/main/scala/com/cognite/sdk/scala/v1/sequenceRows.scala

```diff
@@ -1,8 +1,9 @@
 package com.cognite.sdk.scala.v1
 
+import com.cognite.sdk.scala.common.ResponseWithCursor
 import io.circe.Json
 
-final case class SequenceColumnId(externalId: String)
+final case class SequenceColumnSignature(externalId: String, name: Option[String], valueType: String)
```

Comment:
`valueType` is optional, with a default value of "DOUBLE". We could make it an option as well, but I guess it's nicer if we do `valueType: String = "DOUBLE"`.

Reply:
If we're adding fields here maybe we should also add `description` and `metadata`?

Reply:
Ok, I'll make that optional. It seems that the API is sending it in all cases, though.

> If we're adding fields here maybe we should also add description and metadata?

These are not present in this context, however.

![image](https://user-images.githubusercontent.com/7894687/81717146-b6f1d380-9469-11ea-845d-cf41870bb66d.png)

Reply:
Nevermind, I had it confused with `SequenceColumnCreate`, you can remove the default for `valueType`. :)